CODEOWNERS

```diff
@@ -6,4 +6,5 @@ CODEOWNERS @microsoft/configmgr-hub
 README.md @microsoft/configmgr-hub
 * @microsoft/configmgr
 /tools/ExtensionDownloader.ps1 @microsoft/configmgr-hub
-/pipelines/secure-scan.yml @microsoft/configmgr-hub
\ No newline at end of file
+/pipelines/secure-scan.yml @microsoft/configmgr-hub
+/object/ConsoleExtension/"Right Click Tools.json" @recast-software/community-hub
```

Comment:
I'm not seeing that the recast\community-hub team is a required reviewer.... Wondering if this needs to actually exist FIRST in the branch or if out of organization teams don't work for CODEOWERS?

Reply:
@bryand-recastsoftware  Maybe once we are all signed off and ready for submission one of us can try to complete the PR to verify this behavior. Want to make sure we have the right controls in place for Extension owners so you are in control.

---
In reply to: [610234166](https://github.com/microsoft/configmgr-hub/pull/101#discussion_r610234166) [](ancestors = 610234166)

Reply:
My assumption is also it has to be on master branch already for it to take effect.

Reply:
@silveyy: Do you want me to submit a separate PR to master for CODEOWNERS that the team approves and merges first?

Reply:
Except the file /object/ConsoleExtension/"Right Click Tools.json" doesn't exists yet on the separate PR, but may be it will still work.

Reply:
That or we can read the manual... on second thought let's do that. :)

---
In reply to: [610239296](https://github.com/microsoft/configmgr-hub/pull/101#discussion_r610239296) [](ancestors = 610239296)

Reply:
And here I thought the plan was to just keep failing upwards.  Man, you guys and your 'reading'. 'Manuals'. Overachievers the whole the lot of you!